src/App.tsx

```diff
@@ -1,61 +1,93 @@
 import './App.scss';
-
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Todo } from './react-app-env';
 
 export const App = () => {
+  const [userSelect, setUserSelect] = useState<number>(-1);
```

Comment:
The type is set automatically

```suggestion
  const [userSelect, setUserSelect] = useState(-1);
```

Reply:
pay attention to this comment